connect/evidence/containers.py

```diff
@@ -91,6 +91,36 @@ def _validate(self, data):
             )
 
 
+class StatisticTestContainer(EvidenceContainer):
+    """Containers for all Statistic Test type evidence"""
+
+    def __init__(self, data: pd.DataFrame, labels: dict = None, metadata: dict = None):
+        super().__init__(StatisticTestEvidence, data, labels, metadata)
+
+    def to_evidence(self, **metadata):
+        evidence = []
+        for _, data in self.scrubbed_data.iterrows():
+            evidence.append(
+                self.evidence_class(
+                    additional_labels=self.labels, **data, **self.metadata, **metadata
+                )
+            )
+        return evidence
+
+    def _validate(self, data):
+        required_columns = {
+            "test_statistic",
```

Comment:
Cofnusing labeling for me. The `test_statistic` is the value - it's a quantitative thing. I'd just use "type" for the name, similar to MetricEvidence.
```suggestion
            "type",
            "test_statistic",
            "significance_threshold",
            "p_value"
```